libsolidity/formal/SMTEncoder.cpp

```diff
@@ -274,6 +274,20 @@ bool SMTEncoder::visit(InlineAssembly const& _inlineAsm)
 	return false;
 }
 
+bool SMTEncoder::visit(TryCatchClause const& _try)
+{
+	if (auto params = _try.parameters())
+		for (auto const& var: params->parameters())
```

Comment:
This is where I actually don't like auto as type is not obvious. But as codebase is mixed with both using auto and actual type and there are no rules I guess it is fine to use auto :)

Reply:
To me this is exactly where auto is nice, and IMO it makes sense for range loops. The type is not written there, but it's inferrable from the name.